docs/content/server/new_blocks.md

```diff
@@ -147,3 +147,67 @@ This approach allows us to test the block's logic comprehensively without relyin
 6. **Update tests when changing block behavior**: If you modify your block, ensure the tests are updated accordingly.
 
 By following these steps, you can create new blocks that extend the functionality of the AutoGPT Agent Server.
+
+## Blocks we want to see
+
+Below is a list of blocks that we would like to see implemented in the AutoGPT Agent Server. If you're interested in contributing, feel free to pick one of these blocks or suggest your own by editing [docs/content/server/new_blocks.md](https://github.com/Significant-Gravitas/AutoGPT/edit/master/docs/content/server/new_blocks.md) and opening a pull request.
+
+### Consumer Services/Platforms
+
+- Google sheets - Read/Append [Read in Progress](https://github.com/Significant-Gravitas/AutoGPT/pull/7521)
+- Email - Read/Send with Gmail, Outlook, Yahoo, Proton, etc
+- Calendar - Read/Write with Google Calendar, Outlook Calendar, etc
+- Home Assistant - Call Service, Get Status
+- Dominos - Order Pizza, Track Order
+- Uber - Book Ride, Track Ride
+- Notion - Create/Read Page, Create/Append/Read DB
+- Google drive - read/write/overwrite file/folder
+
+### Social Media
+
+- Twitter - Post, Reply, Get Replies, Get Comments, Get Followers, Get Following, Get Tweets, Get Mentions
+- Instagram - Post, Reply, Get Comments, Get Followers, Get Following, Get Posts, Get Mentions, Get Trending Posts
+- TikTok - Post, Reply, Get Comments, Get Followers, Get Following, Get Videos, Get Mentions, Get Trending Videos
+- LinkedIn - Post, Reply, Get Comments, Get Followers, Get Following, Get Posts, Get Mentions, Get Trending Posts
+- YouTube - Transcribe Videos/Shorts, Post Videos/Shorts, Read/Reply/React to Comments, Update Thumbnails, Update Description, Update Tags, Update Titles, Get Views, Get Likes, Get Dislikes, Get Subscribers, Get Comments, Get Shares, Get Watch Time, Get Revenue, Get Trending Videos, Get Top Videos, Get Top Channels
+- Reddit - Post, Reply, Get Comments, Get Followers, Get Following, Get Posts, Get Mentions, Get Trending Posts
+- Treatwell (and related Platforms) - Book, Cancel, Review, Get Recommendations
+- Substack - Read/Subscribe/Unsubscribe, Post/Reply, Get Recommendations
+- Discord - Read/Post/Reply, Moderation actions
+- GoodReads - Read/Post/Reply, Get Recommendations
+
+### E-commerce
+
+- Airbnb - Book, Cancel, Review, Get Recommendations
+- Amazon - Order, Track Order, Return, Review, Get Recommendations
+- eBay - Order, Track Order, Return, Review, Get Recommendations
+- Upwork - Post Jobs, Hire Freelancer, Review Freelancer, Fire Freelancer
+
+### Data/Information
+
+- Summarize top news of today, of this week, this month via apple news or other large media outlets BBC, techcrunch, hackernews etc

```

Comment:
I would be careful with things like this as they might confuse people as to what a block should be. This feels more like an agent if it's creating a summary of the news over a period of time.

Reply:
Oh yeah good point. Fully agree